control/plugin/cpolicy/node.go

```diff
@@ -78,13 +78,9 @@ func (c *ConfigPolicyNode) UnmarshalJSON(data []byte) error {
 	if err := decoder.Decode(&m); err != nil {
 		return err
 	}
-	if n, ok := m["PolicyNode"]; ok {
-		if pn, ok := n.(map[string]interface{}); ok {
-			if rs, ok := pn["rules"]; ok {
-				if rules, ok := rs.(map[string]interface{}); ok {
-					addRulesToConfigPolicyNode(rules, c)
-				}
-			}
+	if rs, ok := m["rules"]; ok {
```

Comment:
Do you think it's better to put "rules" as a global constant so that all code references the same instance? If any change in one place, all is changed.